src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -422,6 +435,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configRosetta = False
     , _configServiceApi = defaultServiceApiConfig
     , _configOnlySyncPact = False
+    , _configSyncPactChains = []
```

Comment:
Does this mean that `[]` as the same meaning as providing all chain ids? I wonder if that's confusing. I would expect "unset" to be encoded as `Nothing` or to use all chains as default.

Reply:
It does. I think it makes sense to use `Nothing` as default here as well, IIRC I tried that at some point and didn't like it for some reason, I'll try again.